Sources/SwiftGenKit/Parsers/Strings/StringsParser.swift

```diff
@@ -29,20 +29,25 @@ public enum Strings {
   }
 
   private enum Option {
```

Comment:
Wondering if we shouldn't keep that `public`, so that other apps using SwiftGenKit from code could do:
```
let parser = Strings.Parser(options: [Strings.Parser.Option.separator.key: "__"])
```
to allow them to avoid hard-coded strings in their code… (well, if we do that, we might want to check if some other properties on `AnyParserOption` & co would also need to be public too)